orchagent/muxorch.cpp

```diff
@@ -1156,7 +1156,7 @@ MuxOrch::MuxOrch(DBConnector *db, const std::vector<std::string> &tables,
          neigh_orch_(neighOrch),
          fdb_orch_(fdbOrch)
 {
-    handler_map_.insert(handler_pair(CFG_MUX_CABLE_TABLE_NAME, &MuxOrch::handleMuxCfg));
+    handler_map_.insert(handler_pair(CFG_LINKMGR_CABLE_TABLE_NAME, &MuxOrch::handleMuxCfg));
```

Comment:
This would impact multiple places even in the VS tests, Sonic-mgmt tests etc and also debug efforts. what's the critical motivation for such an impacting change?

Reply:
The link manager related tables in config db all have `mux` prefix, the PR: https://github.com/Azure/sonic-swss-common/pull/580 changes the table name to be more generic.